admin_manual/configuration/server/caching_configuration.rst

```diff
@@ -183,6 +183,24 @@ The ``yum`` command shows available and installed version information:
 
    yum search php-pecl-redis
 
+Clearing the Redis Cache
```

Comment:
Isn´t it necessary to need to create a link reference here respectively on Memcache below so it can be directly linked by #3687 ?

Reply:
No, because headers are able to be linked to because each one is an automatic anchor.

Reply:
great, did not know that... 😄